gethrpc/Cargo.toml

```diff
@@ -5,10 +5,10 @@ license = "Apache-2.0"
 authors = ["Stewart Mackenzie <setori88@gmail.com>", "Wei Tang <hi@that.world>"]
 description = "gethrpc - provides functionality for interacting with geth's blockchain."
 repository = "https://github.com/etclabscore/sputnikvm"
+edition = "2018"
 
 [dependencies]
 clap = "2.22"
-serde = "1.0"
-serde_derive = "1.0"
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
Just a note this is backward incompatible for anyone requiring this library. It breaks all packages depending on this crate that do this before:

```
use serde::{Serialize, Deserialize};
use serde_derive::{Serialize, Deserialize};
```

Reply:
@sorpaas I'm not sure that's correct, as `gethrpc` doesn't re-export `Serde` types, which are stable since `serde 1.0`. So the users should be able to use `serde_derive` or `derive feature` interchangeably without problems for their types. And serialization crates as `serde_json` would work with `gethrpc` types too without any changed necessary.
Can you share some link? I might be missing something.

Anyway, this is not an issue since these changes would go into the next major version `0.11`.